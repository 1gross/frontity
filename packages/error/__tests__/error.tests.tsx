import { FrontityError } from "..";

describe("FrontityError", () => {
  test("should output an error message in the correct format", () => {
    const error = new FrontityError("Something bad happend!");

    expect(error.toString()).toMatchSnapshot();
    expect(() => {
      throw error;
    }).toThrowErrorMatchingSnapshot();
  });
});
