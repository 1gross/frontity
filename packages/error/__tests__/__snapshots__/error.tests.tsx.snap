// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FrontityError should output an error message in the correct format 1`] = `
"FrontityError: Something bad happend!

If you have any doubts, join our community at https://community.frontity.org/.
"
`;

exports[`FrontityError should output an error message in the correct format 2`] = `
"Something bad happend!

If you have any doubts, join our community at https://community.frontity.org/.
"
`;
